spec/models/order_spec.rb

```diff
@@ -1,5 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe Order, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  context '#orders' do
+    it 'has a user id' do
+      order = create_order
+
+      expect(order.user_id).to eq(2)
+    end
+
+    it 'has user trips' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.